neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
Tohle bych chtěl mergovat zvlášť i z toho důvodu, že to podstatně mění funkcionalitu, zatímco následný refaktor celého modulu by měl být zpětně zcela kompatibilní.